src/ReceiveCommands.cc

```diff
@@ -3150,12 +3150,16 @@ shared_ptr<Lobby> create_game_generic(
   bool item_tracking_enabled =
       (c->version() == GameVersion::BB) ||
       (s->item_tracking_enabled && (mode == GameMode::NORMAL || mode == GameMode::SOLO));
+  
+  bool drops_enabled =
+      (s->drops_enabled && (mode == GameMode::NORMAL));

```

Comment:
Enabled drops should be the default behavior, right? Presumably we want `(s->drops_enabled || (mode != GameMode::NORMAL))` here.